### DISCUSSION TITLE
Hangar High Rounds

### DISCUSSION THREAD

**Discussion by itzxil** [nzp-team/nzportable#594] [Records and Speedruns]
This game is sorta old at this point but i just realized i never posted it here
![Screenshot 2024-01-10 092402](https://github.com/nzp-team/nzportable/assets/51383442/6415acb4-2b45-4242-a651-7df4b1664f2a)

https://www.youtube.com/playlist?list=PLHkCcjsGVK8rjdyE-PbaMXRRXBkqRhnj2

**Comment by Vinxent9** [comment]
nicely done ^_^b

**Comment by architsasha45-afk** [comment]
what the hell
